### ISSUE TITLE
No data in S3 or DDB

### ISSUE THREAD

**Issue by SaadKhurshid1** [amazon-connect/amazon-connect-realtime-transcription#38]
I have followed the "Easy Setup" and created Cloud Formation Stack.
But i don't see any data in DynamoDB or S3.

What could i be missing here? Any idea?

Thanks

**Comment by SaadKhurshid1** [comment]
Following info is present in the logs for KvsTranscriber function

2020-03-06 09:12:07 INFO  KVSTranscribeStreamingLambda:159 - Calling Transcribe service..
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingsoftware.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingsoftware.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingsoftware.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingsoftware.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingsoftware.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingsoftware.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 INFO  TranscribeStreamingRetryClient:214 - Reached on error but doing nothingjava.util.concurrent.CompletionException: software.amazon.awssdk.core.exception.SdkClientException
2020-03-06 09:12:10 DEBUG TranscribeStreamingRetryClient:164 - Error occurred on channel AUDIO_TO_CUSTOMER : software.amazon.awssdk.core.exception.SdkClientException
java.util.concurrent.CompletionException: software.amazon.awssdk.core.exception.SdkClientException
	at software.amazon.awssdk.utils.CompletableFutureUtils.errorAsCompletionException(CompletableFutureUtils.java:61) ....

**Comment by marialinkeit** [comment]
Same issue here, any updates on the matter?

**Comment by SaadKhurshid1** [comment]
no response yet.. have you figured it out? @marialinkeit

**Comment by marialinkeit** [comment]
Yes, @SaadKhurshid1 , I did!
Turns out Transcribe and Connect don't work in the same regions, and I was using Connect (and the lambdas) in Frankfurt, where Transcribe is not available, so I had to override the region for the Transcribe service to Ireland. To do this, there is an environment variable in the kvsTranscriber function, the Java one, called TRANSCRIBE_REGION. Just set it to eu-west-1 and it worked! 

I found the answer in this closed issue: #25 

Sorry for not updating!
